core/src/test/scala/cats/derived/show.scala

```diff
@@ -84,9 +83,25 @@ class ShowSuite extends KittensSuite {
 //    println(Show[Tree[Int]])
   }
 
-  semiTests.run()
+  {
+    import semiInstances._
+    testShow("semi")
+  }
+}
+
+object ShowSuite {
+  import TestDefns._
+
+  implicit val showAddress: Show[Address] = Show.show { a =>
+    List(a.street, a.city, a.state).mkString(" ")
+  }
+
+  final case class Bogus(value: Int)
+  object Bogus {
+    implicit val show: Show[Bogus] = Show.show(_ => "Blah")
+  }
 
-  object semiTests {
+  object semiInstances {
```

Comment:
I'm not sure why I had to move these instances to make them serializable. Looks like somehow `this` was captured in a closure (maybe by `Typeable` because only `Show` and `ShowPretty` are affected, and that on Scala 2.12).